projects/demo/src/modules/components/primitive-textfield/examples/2/index.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    ChangeDetectorRef,
+    Component,
+    Inject,
+    Optional,
+    Self,
+    ViewChild,
+} from '@angular/core';
+import {NgControl} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {AbstractTuiControl, TuiNativeFocusableElement} from '@taiga-ui/cdk';
+import {
+    TUI_PRIMITIVE_TEXTFIELD_DEFAULT_OPTIONS,
+    TUI_PRIMITIVE_TEXTFIELD_OPTIONS,
+    TuiPrimitiveTextfieldComponent,
+} from '@taiga-ui/core';
+
+@Component({
+    selector: 'tui-primitive-textfield-example-2',
+    templateUrl: './index.html',
+    changeDetection,
+    encapsulation,
+    providers: [
+        {
+            provide: TUI_PRIMITIVE_TEXTFIELD_OPTIONS, // <-- You are looking for this token
```

Comment:
nit: redundant comment

Reply:
I definitely agree with your point but for the demo purposes everything else in this file is just to get the component up&running and I thought it'd be nice to point out the exact piece of code related to this demo. Anyway, if you'd like me to remove this, I'll do that by this evening

Reply:
I think we can keep it, there's a lot of code in this example and it might be helpful to point to the important part.